electron-app/src/services/rotkehlchen-api.ts

```diff
@@ -184,6 +184,46 @@ export class RotkehlchenApi {
     });
   }
 
+  changeUserPassword(
+    username: string,
+    currentPassword: string,
+    newPassword: string
+  ): Promise<boolean> {
+    return new Promise<boolean>((resolve, reject) => {
```

Comment:
Promise wrapping is redundant, I actuially need to go an remove it everywhere. look towards the end of the file.

Reply:
Ah yes you had mentioned that before, will look into it!